### PR TITLE
Restore hello bar

### PR DESCRIPTION
**Pull request by BradDenver**
- replaces bsa fancy bar with hello bar again

### CODE REVIEW

File: assets/javascripts/discourse/initializers/random-hello-bar.js.es6 [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+        var isEnabled = true;
+
+        // if jQuery Cookie is available, we may have disabled ourselves for the number of days set in EXPIRE_DAYS if user manually closed the bar
+        if ($.cookie) {
+          if ($.cookie('awesomeBarDisabled')) {
+            isEnabled = false;
+          }
+        }
+
+        if ($('html').hasClass('mobile-device')) isEnabled = false;
```
Comment (BradDenver) [inline assets/javascripts/discourse/initializers/random-hello-bar.js.es6]: no longer loads on mobile devices (same as bsa version)

**Comment by BradDenver** [comment]
This is a quick PR to swap out BSA fancy bar for the hello bar as it has partnerships have now sold this ad space.

I will be be committing a refactor of the JS and CSS next.

**Comment by kellec** [comment]
Sure :+1:

**Comment by santouras** [comment]
:+1:

**Pull request merged**